common/protos/multiboot2.c

```diff
@@ -314,24 +314,24 @@ noreturn void multiboot2_load(char *config, char* cmdline) {
         }
     }
 
-    uint64_t reloc_slide = 0;
+    int64_t reloc_slide = 0;
 
     if (has_reloc_header) {
         bool reloc_ascend;
-        uint64_t relocated_base;
+        int64_t relocated_base;
```

Comment:
I wouldn't change this to signed. As I said previously I'd cast it to `int64_t` on line 359.